### PR TITLE
Update Nations.json

### PR DESCRIPTION
**Pull request by Smashfanful**

### CODE REVIEW

**Comment by yairm210** [comment]
I don't understand what this is about.

**Comment by krushia** [comment]
The nations have been rearranged to be in alphabetical order, which is why the diff is hard to follow.

**Comment by Smashfanful** [comment]
Yeah, I should have added that description sooner. Also, Sweden has been removed because not relevant to Vanilla at all.

**Comment by yairm210** [comment]
I really don't think this is necessary. I don't think it adds anything. You can just search for the civ with ctrl+f.
